src/Commands/Env/ViewCommand.php

```diff
@@ -16,21 +16,21 @@ class ViewCommand extends TerminusCommand implements SiteAwareInterface, Contain
     use ContainerAwareTrait;
 
     /**
-     * Print the URL for an environment or open it in a browser
+     * Displays the URL for the environment or opens the environment in a browser.
      *
      * @authorize
      *
      * @command env:view
      * @aliases site:view
      *
-     * @param string $site_env The site and environment to view in the form: <sitename>.<env>
-     * @option boolean $print Output the URL only. Do not open the URL in the default browser.
+     * @param string $site_env Site & environment in the format `site-name.env`
+     * @option boolean $print Print URL only
      * @return string
      *
-     * @usage: terminus env:view <site>.<env> --print
-     *    Outputs the URL of the <env> environment of <site>
      * @usage: terminus env:view <site>.<env>
-     *    Opens the URL of the <dev> environment of <site> in your default browser
+     *    Opens browser to <site>'s <env> environment.
```

Comment:
I've been trying to do away with definite articles. but can put `the` in it (really trying not to use `your`)